ipython_profiles/profile_s6bm/startup/03-plans.py

```diff
@@ -218,3 +221,13 @@ def repeat_exp(plan_func, n=1):
 
 
 print(f'leaving {__file__}...\n')
+
+
+def repeat(n): 
```

Comment:
Suggest replacing this code with [`bluesky.plan_stubs.repeat()`](https://github.com/bluesky/bluesky/blob/59773b474fa7a5688bea3582313dfe2946ee3a49/bluesky/plan_stubs.py#L955)

Reply:
This `repeat()` function is really neat for repeating single scan.  However, is it possible to extend its capability by adding a `timeout` option?